azurelinuxagent/common/protocol/wire.py

```diff
@@ -876,6 +930,37 @@ def check_wire_protocol_version(self):
                      "advised by Fabric.").format(PROTOCOL_VERSION)
             raise ProtocolNotFoundError(error)
 
+    def send_request_hostplugin_first(self, direct_func, host_func):
+        # For certain calls we want them to go through the HostGAPlugin first, since it
+        # supports the If-None-Match header that reduces how often we need to parse
+
+        # note that this method requires a lambda that returns two values
+        ret = None
+        etag = None
+        not_modified = False
```

Comment:
In this case, I'd really like to keep the not_modified name because it specifically means that a 304 NOT MODIFIED response was returned from the server. So here, calling it "modified" would be more confusing.

Reply:
Ahh ok ok that makes sense, could you add a small comment then please? Just stating that this means a 304 response kinda thing to make it a bit clearer when reading the code. Thanks!

Reply:
added a comment in my most recent push